fontforge/svg.c

```diff
@@ -3145,15 +3184,20 @@ return;
 }
 
 static SplineFont *SVGParseFont(xmlNodePtr font) {
-    int cnt, flags = -1;
+    int cnt;
     xmlNodePtr kids;
     int defh=0, defv=0;
     int has_font_face = false;
     xmlChar *name;
     SplineFont *sf;
     EncMap *map;
+    ImportParams *ip = ImportParamsState();
+    bigreal tmpjl = ip->default_joinlimit;
     int i;
 
+    if ( tmpjl==JLIMIT_INHERITED )
```

Comment:
I don't know how to feel about this approach, it seems brittle/easy to get wrong

Reply:
I didn't want to have different miter/join limits for different formats, but still wanted it to be possible for each format to have its corresponding default when that was desired. This seemed like the most straightforward means of doing that.

Reply:
Yeah, just seems kind of dodgy to have to modify then unwind the change at the end of the call. I can imagine this would pretty easily break the next time someone comes along and returns before resetting the value.

Not necessarily for now, but something to think about - how about a separate field that defines the default value?  Then if the value is `inherited`, you use that, otherwise you use the normal value. Or maybe somewhat cleaner - you could also define a 'default type' field or something similar (i.e. indicate this is for SVG/etc.), then have the defaults defined in one place.